torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1152,9 +1152,19 @@ at::Tensor XLANativeFunctions::expand_symint(const at::Tensor& self,
                                              at::SymIntArrayRef sym_size,
                                              bool implicit) {
   XLA_FN_COUNTER("xla::");
-  auto size = c10::asIntArrayRefSlow(sym_size);
-  return bridge::AtenFromXlaTensor(XLATensor::expand(
-      bridge::GetXlaTensor(self), torch::lazy::ToVector<int64_t>(size)));
+  c10::optional<at::IntArrayRef> size = c10::asIntArrayRefSlowOpt(sym_size);
+  if (size.has_value()) {
```

Comment:
It seems `size.has_value()` implies none of the dimensions is symbolic, right?

Reply:
right, then we can fall back to static expand

Reply:
yes, it's a pretty common idiom in PyTorch for asXXX to return std::nullopt if a conversion was unsuccessful